plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -480,6 +481,10 @@ public StorageSubsystemCommandHandler getStorageHandler() {
         return storageHandler;
     }
 
+    public Properties getProperties() {
```

Comment:
Is this method used somewhere? Can it be renamed to `getUefiProperties`?

Reply:
changes done as suggested.